src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -291,7 +291,7 @@ internal override bool HasMetadataConstantValue
         {
```

Comment:
@agocke I agree it would have the same behavior, but a completely different performance profile. The override serves to avoid attempting to unpack the (constant value) attributes when the HasDefault flag is not set. I believe this is a significant performance benefit.